spec/support/shared_examples/hyrax_basic_metadata.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'Hyrax Basic Metadata Create', js: false do
+  let(:admin) { FactoryBot.create(:admin) }
+  let(:etd) do
+    FactoryBot.create(
```

Comment:
Rather than use `create`, which round-trips to Fedora, we can do `FactoryBot.attributes_for` here to just get the attributes without any database interactions. It doesn't look like we use the `etd` object for anything other than accessing its attributes, right?

Reply:
That's true.